build.gradle

```diff
@@ -65,9 +65,9 @@ sharedLibrary {
     pluginDependencies {
         // https://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/workflow/workflow-cps-global-lib/
         // Need to migrate to pipeline-groovy-lib later as workflow-cps-global-lib is deprecated now.
-        workflowCpsGlobalLibraryPluginVersion = '588.v576c103a_ff86'
+        workflowCpsGlobalLibraryPluginVersion = '613.v9c41a_160233f'
```

Comment:
I didn't see this version in https://repo.jenkins-ci.org/artifactory/public/org/jenkins-ci/plugins/workflow/workflow-cps-global-lib/. I believe this caused the groovy failure. This library was deprecated so we might need to migrate to `pipeline-groovy-lib`

Reply:
The shared-library extension that we are using in build.gradle has not been updated since 2020. https://github.com/mkobit/jenkins-pipeline-shared-libraries-gradle-plugin

Not sure what the `pipeline-groovy-lib` version of `workflowCpsGlobalLibraryPluginVersion` will be called.

Reply:
Made the change